site/api/prisma/schema.prisma

```diff
@@ -116,10 +116,25 @@ model Alias {
   @@map("aliases")
 }
 
+model PersonNationality {
```

Comment:
See underneath.. I wish this would be called Idol instead of Person..
Or artist